batch/post_process_data_analysis.py

```diff
@@ -0,0 +1,624 @@
+"""
+Script to analyze and visualze gauge errors for the batch runs
+"""
+# ============================================================================
+#      Copyright (C) 2013 Kyle Mandli <kyle@ices.utexas.edu>
+#
+#          Distributed under the terms of the MIT license
+#                http://www.opensource.org/licenses/
+#
+# (post process scripts by Akshay Sriapda <as4928@columbia.edu>, 2017)
+# ============================================================================
+
```

Comment:
Same goes here.  Probably should remove this from the other source files (old standard).